src/ert/storage/migration/block_fs.py

```diff
@@ -314,6 +314,7 @@ def migrate_case(storage: Storage, path: Path, stack: ExitStack) -> None:
         parameters=parameter_configs,
         responses=response_configs,
         observations=ert_config.observations,
+        name="migrate-case",
```

Comment:
Should this be made unique or is it ok for cases to be migrated (from blockfs in this case so maybe no that many) w/ the same experiment name? Logging statement above is
`    logger.info(f"Migrating case '{path.name}'")`

Reply:
Valid point, but this not change behavior as the current code does this when no name is specified:

`name = datetime.today().strftime("%Y-%m-%d")`

Will leave as is for now. OK?